
![mercury](https://github.com/user-attachments/assets/75d49c42-e83c-4e13-a902-60c44824baac)
Abstract
This study examined how soil mercury contamination affected the structure and functionality of rhizobacteria communities from Aeschynomene fluminensis and Polygonum acuminatum and how rhizobacteria mediate metal bioremediation. The strains were isolated using culture-dependent methods, identified through 16S rDNA gene sequencing, and characterized with respect to their functional traits related to plant growth promotion and resistance to metals and antibiotics. The bioremediation capacity of the rhizobacteria was determined in greenhouse using corn plants. The isolated bacteria belonged to the phyla Actinobacteria, Deinococcus-Thermus, Firmicutes, and Proteobacteria, with great abundance of the species Microbacterium trichothecenolyticum. The rhizobacteria abundance, richness, and diversity were greater in mercury-contaminated soils. Bacteria isolated from contaminated environments had higher minimum inhibitory concentration values, presented plasmids and the merA gene, and were multi-resistant to metals and antibiotics. Enterobacter sp._C35 and M. trichothecenolyticum_C34 significantly improved (Dunnett's test, p < 0.05) corn plant growth in mercury-contaminated soil. These bacteria helped to reduce up to 87% of the mercury content in the soil, and increased the mercury bioaccumulation factor by up to 94%. Mercury bioremediation mitigated toxicity of the contaminated substrate. Enterobacter sp._C35, Bacillus megaterium_C28, and Bacillus mycoides_C1 stimulated corn plant growth and could be added to biofertilizers produced in research and related industries.
Keywords: Bioaccumulation; Bioremediation; Enterobacter; Microbacterium; Rhizoremediation; Toxic metal.

All the data analyses were performed using the software R 3.4.2 (R Development Core Team 2016). Bacterial counts in the host plant rhizosphere were expressed in CFU per gram of dry soil (CFU g−1) and compared using the non-parametricKruskal-Wallis test. The diversity profile of the communities was analyzed through the Hill Series, using the vegan package. The community dissimilarity was estimated by cluster analysis using the Bray-Curtis dissimilarity matrix with abundance data from the four bacterial communities (+AHg, −AHg, + PHg, and −PHg). The species composition of the communities was visualized in the Venn diagram constructed using the Online Software(http://bioinformatics.psb.ugent.be/webtools/Venn/). The similarity between the communities’ structure according to their functional traits was analyzed through the Jaccard dissimilarity index. Results from the qualitative functional characterization were expressed as positive (+) and negative (−) when a given functional trait was present or absent, respectively. Non-metric multidimensional scaling using the Bray-Curtis index was performed withthe metaMDS and Adonis function of vegan package. The indicator species were verified using the labdsv package.Co-occurrence analysis was performed using the Spearman correlation of the hmisc and corrplot packages. In the plant growth promotion and bioremediation experiments, the treated samples were compared with the control plants grown on substrates contaminated (C+Hg) or not with mercury (C−Hg), using analysis of variance followed by Dunnett’s test, with the aid of the multcomp package.
